multichain-node/entrypoint.sh

```diff
@@ -23,9 +32,9 @@ rpcallowip=$RPC_ALLOW_IP
 rpcport=$RPC_PORT
 EOF
 
-echo "Setup /data/explorer.conf"
+echo "Setup /root/explorer.conf"
```

Comment:
De `datadir` is nog steeds `/data` Moet dit dan wel `/root` zijn?

Reply:
Ik had er over nagedacht, en het kan beide. Omdat het niet bij de chain hoort, vond ik het beter om het hier te plaatsen. Dat alleen de chain-data gebackupt wordt. Dit zijn de credentials die de explorer web-interface gebruikt om met multichain te verbinden.